java/com/engflow/notificationqueue/README.md

```diff
@@ -0,0 +1,65 @@
+# Using the notification queue and event store APIs
+
+This documents explains how to make use of the **notification queue API** to
+obtain the notifications for a given build and use the notification data
+to obtain invocation events from the **event store API**. To do so,
+the [EngFlow API](https://github.com/EngFlow/engflowapis) is used as external
+dependency (see [WORKSPACE](../../../../WORKSPACE)).
+
+# Running the client
+
+In order to execute the client example, you must pass the following arguments
+that depends on the target cluster. 
+
+1. `--notification_queue_endpoint=CLUSTER_URL` holds the `grpc/grpcs` endpoint of the cluster
```

Comment:
Rephrase: the URL of the cluster gRPC server. Must start with `grpc://` or `grpcs://`.